openmetrics/src/main/java/io/airlift/openmetrics/MetricsResource.java

```diff
@@ -236,6 +241,15 @@ private String inferAttributesForObjectName(ObjectName objectName)
         return expositions.toString();
     }
 
+    private static List<Map.Entry<String, String>> getLabels(ObjectName objectName)
+    {
+        List<Map.Entry<String, String>> labels = Stream.concat(
```

Comment:
It might be simpler and more efficient to just generate an immutablemap with an immutablemap.builder in a loop here. That said, again, we don't want to tie ourselves to jmx here.